tasks/create_record.json

```diff
@@ -1,16 +1,16 @@
 {
     "puppet_task_version": 1,
     "supports_noop": false,
-    "description": "Create a ServiceNow object",
+    "description": "Create a ServiceNow record\nbolt task run --nodes dev84270 servicenow_tasks::create_record table=incident fields='{\"description\":\"An example incident.\"}'",
     "files": ["ruby_task_helper/files/task_helper.rb"],
     "remote": true,
     "parameters": {
         "table": {
             "description": "ServiceNow table",
             "type": "Optional[String[1]]"
         },
-        "data": {
-            "description": "ServiceNow data",
+        "fields": {
+            "description": "ServiceNow fields",
             "type": "String[1]"
```

Comment:
Ah just noticed that this is a String. This should probably be a hash-type of <field> => <value>. You could probably update the description to include that.

Reply:
Also, I think the description for fields could be `ServiceNow fields. This is a hash of <field_name> => <value>`.

Reply:
fixed